ansible/roles/ocp-workload-migration/defaults/main.yml

```diff
@@ -7,5 +7,17 @@ migration_workload_title: "{{ 'Creating' if not migration_workload_destroy else
 migration_workload_state: "{{ 'present' if not migration_workload_destroy else 'absent' }}"     # state of k8s resources
 silent: false
 
+mig_download_content:
+- url: "https://raw.githubusercontent.com/konveyor/mtc-breakfix/master/02-Stage-Pods/02-stage-pod.sh"
```

Comment:
Is master a good idea? What if they update things? Are there tags you could use?

Reply:
@wkulhanek I think master should be fine. I maintain that repo too. We aren't using tags currently.